From cc0e2a7b162eea6f7918dd29cfbbcd96aaa59a09 Mon Sep 17 00:00:00 2001
From: Jeremy Huddleston Sequoia <jeremyhu@apple.com>
Date: Sat, 17 Jan 2015 17:56:44 -0800
Subject: [PATCH 1003/1012] MacPorts Only: Fix libclang_rt dylib ids

We should probably not be shipping these.

Signed-off-by: Jeremy Huddleston Sequoia <jeremyhu@apple.com>
(cherry picked from commit 1e79aa1d2d652fe79c244c185eeff76c2e40caf2)
---
 runtime/compiler-rt/Makefile | 1 +
 1 file changed, 1 insertion(+)

diff --git llvm_release_34/tools/clang/runtime/compiler-rt/Makefile macports_release_34/tools/clang/runtime/compiler-rt/Makefile
index c6f38b2..273e482 100644
--- llvm_release_34/tools/clang/runtime/compiler-rt/Makefile
+++ macports_release_34/tools/clang/runtime/compiler-rt/Makefile
@@ -235,6 +235,7 @@ $(PROJ_resources_lib)/$1/libclang_rt.%.dylib: \
 		$(ResourceLibDir)/$1/libclang_rt.%.dylib | $(PROJ_resources_lib)/$1
 	$(Echo) Installing compiler runtime library: $1/$$*
 	$(Verb) $(DataInstall) $$< $(PROJ_resources_lib)/$1
+	$(Verb) install_name_tool $$@ -id $$(patsubst $(DESTDIR)%,%,$$@)
 
 # Rule to install runtime libraries.
 RuntimeLibraryInstall.$1: \
-- 
2.10.1

